careers/base/views.py

```diff
@@ -1,7 +1,41 @@
+import json
+import logging
+
 from django.conf import settings
-from django.http import HttpResponse
+from django.http import HttpResponse, HttpResponseBadRequest
+from django.views.decorators.csrf import csrf_exempt
+from django.views.decorators.http import require_POST
+
+from raven.contrib.django.models import client
 
 
 def robots(request):
     action = 'Allow' if settings.ENGAGE_ROBOTS else 'Disallow'
     return HttpResponse('User-agent: *\n{}: /'.format(action), content_type='text/plain')
+
+
+@csrf_exempt
+@require_POST
+def csp_violation_capture(request):
+    data = client.get_data_from_request(request)
+    data.update({
+        'level': logging.INFO,
+        'logger': 'CSP',
+    })
+    try:
+        csp_data = json.loads(request.body)
+    except ValueError:
+        # Cannot decode CSP violation data, ignore
+        return HttpResponseBadRequest('Invalid CSP Report')
+
+    try:
+        blocked_uri = csp_data['csp-report']['blocked-uri']
```

Comment:
Should we capture more info? Looking at the spec, we might also want to at least know the "document-uri" and possibly the "violated-directive".